### ISSUE TITLE
Ref datas no more available in asynchronous mode

### ISSUE THREAD

**Issue by estebangit** [reconquest/atlassian-external-hooks#98]
Bitbucket server: 6.4.0
External Hooks: 7.0.0-1

Hello,
In asynchronous mode, the following script don't receive ref information. It does in synchronous mode.

```
#!/bin/bash

# Dummy script to simply echo parameters

(echo "===================DUMMY========================"
date
echo "Script name: $0"

args=("${@}")

echo "Positional arguments: ${@}"
echo "BB_USER_DISPLAY_NAME: $BB_USER_DISPLAY_NAME"
echo "BB_USER_EMAIL: $BB_USER_EMAIL"
echo "BB_REPO_SLUG: $BB_REPO_SLUG"
echo "BB_REPO_CLONE_SSH: $BB_REPO_CLONE_SSH"

echo "Nb arguments passed: ${#args[@]}"
echo "args: $args"

while read from_ref to_ref ref_name; do
        echo "Ref update:"
        echo "  Old value: $from_ref"
        echo "  New value: $to_ref"
        echo "  Ref name:  $ref_name"
done

echo "Done `date -Iseconds`"
) >>/log/dummy.log
```

The output in sychronous mode :
```
===================DUMMY========================
Wed Sep 11 10:34:04 CEST 2019
Script name: /external-hooks/dummy
Positional arguments: thin
BB_USER_DISPLAY_NAME: xxxxx
BB_USER_EMAIL: xxxx@xxxx.com
BB_REPO_SLUG: dummy
BB_REPO_CLONE_SSH: ssh://git@xxxxx.com:7999/dummy/dummy.git
Nb arguments passed: 1
args: thin
Ref update:
  Old value: 6482bb2cdeb45447235717e6cda8a9f221803622
  New value: f7819f4b76342d94ad1573a2fb28d28add523792
  Ref name:  refs/heads/master
Done 2019-09-11T10:34:04+02:00
```

The output in asychronous mode :
```
===================DUMMY========================
Wed Sep 11 10:34:47 CEST 2019
Script name: /external-hooks/dummy
Positional arguments: thin
BB_USER_DISPLAY_NAME: xxxxx
BB_USER_EMAIL: xxxx@xxxx.com
BB_REPO_SLUG: dummy
BB_REPO_CLONE_SSH: ssh://git@xxxxx.com:7999/dummy/dummy.git
Nb arguments passed: 1
args: thin
Done 2019-09-11T10:34:47+02:00
```

**Comment by kovetskiy** [comment]
Thanks for the bug report, it will be fixed as soon as possible.

**Comment by kovetskiy** [comment]
@estebangit we've just released the fix for the issue, the version is 7.1.0.

https://marketplace.atlassian.com/apps/1211631/external-hooks/version-history#b6332

**Comment by estebangit** [comment]
@kovetskiy Everything is fine. Thanks for the quick release.
